tools/python/strings_txt_from_csv.py

```diff
@@ -0,0 +1,39 @@
+import csv
+from collections import defaultdict
+import sys
+
+resulting_dict = defaultdict(list)
+if len(sys.argv) <= 1:
+    print("""
+       * * *
+
+    This script turns a csv file resulting from "translated strings" in the google sheet file into a strings.txt-formated file.
+
+    To use this script, create the translated strings using the google spread-sheet. Go to file -> Download as, and choose csv. Only the currently open sheet will be exported.
```

Comment:
This text will be formatted by the width of the console. If we brake it into separate lines, the shorter lines might be broken by the console into uneven lines, which will not look good.